README.md

```diff
@@ -1,14 +1,15 @@
 # continuous-build
 
-This is the continuous build repository, with example configurations (and a basic 
+This is an example implementing the continuous building for your repository,
+with example configurations (and a basic
 one set up to run on CircleCI) to show how you can use repo2docker to build
```

Comment:
This is a good idea to emphasize that it's the configurations **and** an actual example, and it's much better said than I originally had put. I might (to the risk of being overly specific) say something like:

> This is an example implementation of a continuous build, which you can easily set up for your own repository! The build here is defined by the configuration file [.circleci/config.yml](.circleci/config.yml) that defines the basics for building on [CircleCI](https://circleci.com/dashboard). Take a look at this file to see how you can use repo2docker to build a container that serves your notebook for others to use.